source/exe/signal_action.h

```diff
@@ -79,8 +82,10 @@ class SignalAction : NonCopyable {
    * This constant array defines the signals we will insert handlers for.
    *
    * Essentially this is the list of signals that would cause a core dump.
+   * The object will contain an array of struct sigactions with the same number
+   * of elements that are in this array.
    */
-  static constexpr int FATAL_SIGS[] = {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGSEGV};
+  static const constexpr int FATAL_SIGS[] = {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGSEGV};
```

Comment:
It is in fact a silly but harmless change, because constexpr implies const for data.  As for the reason:

I had considered making the object contain an std::array with elements corresponding to the signal number, ie, SIGSEGV is the last element in position 11.  This would allow accessing the array of previous signal handlers by indexing using the signal number at the expense of some wasted space.  But to do this the array needed to be sized by the maximum signal number in FATAL_SIGS, and this calculation of maximum signal number needed to be available at compile time.  This led to some interesting code that recursively applied std::max (no loops allowed at compile time until C++14...) and under one of our two compilers seemed to work as constexpr only when the additional const was applied to the FATAL_SIGS.  Ultimately I decided this whole solution was too complex and fragile with only C++11 features and switched to using a local array with one element per fatal signal in the constant array.